02-oop-inheritance/exerciseTopic2.js

```diff
@@ -0,0 +1,98 @@
+class EventEmitter {
+  constructor () {
+    this.listeners = [];
```

Comment:
Change this declaration for an object declaration. In the next methods, you are treating this as an object so you should initialize it as that event thought javascript will do the work in order to make this work.